ci-operator/step-registry/azure/provision/custom-role/azure-provision-custom-role-commands.sh

```diff
@@ -298,6 +298,23 @@ ${required_permissions}
 """
     fi
 
+    #optional permissions for creating a private storage endpoint for internal image registry
```

Comment:
@jinyunma is this something newly introduced for all OCP versions?

Reply:
This is only applied when configuring internal image registry. 
Those permissions have already been listed in [official doc](https://docs.openshift.com/container-platform/4.18/installing/installing_azure/installing-azure-account.html#minimum-required-permissions-ipi-azure_installing-azure-account), but not enable mini permission test for related CI jobs now.

Reply:
got it, thanks.